### ISSUE TITLE
Make JetTricks Unfold Widget Accessible

### ISSUE THREAD

**Issue by eSilverStrike** [CrocoBlock/suggestions#3991]
This issues is similar to #2381

With increased need for WCAG compliance worldwide, it is necessary that buttons be accessible from the keyboard like the one from the Unfold Widget. Currently I can hit tab on the keyboard and select other buttons, links, and menu items on my Elementor page but the unfold button is skipped and cannot be selected by hitting tab on the keyboard 

There may also be other items necessary to make it accessible.

Can you please make the JetTricks plugin accessible.

Thanks!
